src/TrayMenu.ts

```diff
@@ -84,10 +84,10 @@ export default class TrayMenu {
 
   private connectionsMenu() {
     let menu = []
-    const active = this.pool.filter(c => c.active)
-    const recent = this.pool.filter(c => !c.active)
-    if (active.length) menu.push({ label: 'Active connections', enabled: false }, ...this.connectionsList(active))
-    if (active.length && recent.length) menu.push({ type: 'separator' })
+    const connected = this.pool.filter(c => c.connected)
+    const recent = this.pool.filter(c => !c.connected)
```

Comment:
how you know that every device not connected is recent?

Reply:
That's just where we're putting the disconnected connections. A connection object is only created on connection. This logic isn't changing just the variable names.